src/tools/lldump.c

```diff
@@ -151,10 +165,10 @@ main (int argc,
 
 	rtn = 0;
 
-	if (todo.dump_btree)  { dump_index(dbname);   }
 	if (todo.dump_key)    { dump_keyfile(dbname); }
-	if (todo.dump_record) { dump_block(dbname);   }
 	if (todo.dump_xref)   { dump_xref(dbname);    }
+	if (todo.dump_btree)  { dump_index(dbname);   }
+	if (todo.dump_record) { dump_block(dbname);   }
```

Comment:
Would it make sense to dump the blocks before the indexes?

Reply:
I reorganized them from simple to lots of data. putting xref and keys up front, then index (which will be only a few)
and finally all the data.